webpack/react_app/components/FeaturesDropdown/actions.js

```diff
@@ -0,0 +1,13 @@
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import { post } from 'foremanReact/redux/API';
+
+export const runFeature = (hostId, feature, label) => dispatch => {
+  const url = foremanUrl(
+    `/job_invocations?feature=${feature}&host_ids%5B%5D=${hostId}`
```

Comment:
Unfortunatelly this is wrong as we are passing the Host name no an Id here.
That's something I've complained a lot. In core we are passing the host identifier as hostId prop all around the page, but it is usually a name, not an ID, I believe it should be fixed to properly denote what we are actually passing to this prop. @amirfefer could you fix this instance, so it would prefil the search for the job correctly?
@jeremylenz ^ for awarnes about the mess and what it can cause ;)

Reply:
@ezr-ondrej, Yeah, I noticed quite a while back that `host_ids` is named poorly and is actually just a scoped search.  That's the way we treat it in Katello:

```js
`host_ids=name ^ (${hostname})`
```

https://github.com/Katello/katello/blob/2f9e60b313c9cd05f44811db2939ed66fc61e0ac/webpack/components/extensions/HostDetails/Tabs/customizedRexUrlHelpers.js#L13-L14

Let me know if you're going to break this so I can make sure it gets fixed ;)

Reply:
it is not enrirely true, it works now when we made sure, that hostId is a numerical Id :)